[2024-01-02T21:25:22.090+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2023-05-06T00:00:00+00:00 [queued]>
[2024-01-02T21:25:22.099+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2023-05-06T00:00:00+00:00 [queued]>
[2024-01-02T21:25:22.099+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2024-01-02T21:25:22.114+0100] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-05-06 00:00:00+00:00
[2024-01-02T21:25:22.117+0100] {standard_task_runner.py:57} INFO - Started process 87271 to run task
[2024-01-02T21:25:22.126+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2023-05-06T00:00:00+00:00', '--job-id', '14950', '--raw', '--subdir', '/Users/user/Desktop/Learning/AIRFLOW-TUTORIAL/methode-1/dags/dag_with_postgres.py', '--cfg-path', '/var/folders/gn/cr4yj18d1tv34pkws9v_kq1r0000gn/T/tmpmxhe8qdr']
[2024-01-02T21:25:22.129+0100] {standard_task_runner.py:85} INFO - Job 14950: Subtask insert_into_table
[2024-01-02T21:25:22.191+0100] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2023-05-06T00:00:00+00:00 [running]> on host macbook-pro-de-user.local
[2024-01-02T21:25:22.888+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aida' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-05-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-06T00:00:00+00:00'
[2024-01-02T21:25:22.890+0100] {sql.py:274} INFO - Executing: 
            insert into dag_runs(dt, dag_id) values('2023-05-06','dag_with_postgres_operator_v04')
        
[2024-01-02T21:25:22.896+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-01-02T21:25:23.122+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-01-02T21:25:23.155+0100] {sql.py:418} INFO - Running statement: 
            insert into dag_runs(dt, dag_id) values('2023-05-06','dag_with_postgres_operator_v04')
        , parameters: None
[2024-01-02T21:25:23.158+0100] {sql.py:427} INFO - Rows affected: 1
[2024-01-02T21:25:23.166+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20230506T000000, start_date=20240102T202522, end_date=20240102T202523
[2024-01-02T21:25:23.210+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T21:25:23.252+0100] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
